### DISCUSSION TITLE
Inline Creation bug with belongsToMany field not passing viaRelation values

### DISCUSSION THREAD

**Discussion by brenjt** [laravel/nova-issues#4829] [Q&A] [answered]
- Laravel Version: 8.83.14
- Nova Version: 4.15.1
- PHP Version: 8.1.9

### Description:
I am thinking there is a bug with inline relation creation. I have a resource that has a belongsToMany exactly as shown below.
```
BelongsToMany::make('Products')
    ->showCreateRelationButton(),
```

Which generates this view

<img width="815" alt="image" src="https://user-images.githubusercontent.com/1713885/191560323-cd8c8126-77e8-4ad9-9506-72f23dfce9b9.png">

When I click the + button I was expecting the via resource values to be the parent resource that has the BelongsToMany field to be present in the network request, however they are all empty

```
/nova-api/products/creation-fields
?editing=true
&editMode=create
&inline=true
&viaResource=
&viaResourceId=
&viaRelationship=
```

I'd expect the via fields to be set based on the `Form/BelongsToField.vue` but they are empty

```
      <CreateRelationModal
        :show="canShowNewRelationModal && relationModalOpen"
        @set-resource="handleSetResource"
        @create-cancelled="closeRelationModal"
        :resource-name="field.resourceName"
        :resource-id="resourceId"
        :via-relationship="viaRelationship"
        :via-resource="viaResource"
        :via-resource-id="viaResourceId"
      />
```

Am I crazy or is this a bug?

**Comment by crynobone** [comment, marked as answer]
Hi, I don't believe this is a bug. At the point of inline create the relationship is yet to be attached and adding the additional props doesn't 100% useful (since you can create the relationship) and simply cancel attaching.

**Comment by sanderbaas** [comment]
The additional props can be useful to determine which fields one might show or which options are loaded in select fields for instance. Maybe you want to show an extra field if the request is made via a specific relationship.
